assets/i18n/en/database_quests.json

```diff
@@ -5,7 +5,7 @@
   "no_option": "No quest found",
   "new": "New quest",
   "quest_deletion_of": "Deletion of the quest",
-  "quest_deletion_message": "You are about to permanently delete the quest «\u00a0{{quest}}\u00a0».",
+  "quest_deletion_message": "You are about to permanently delete the quest « {{quest}} ».",
```

Comment:
No-break space characters `\u00a0` are important for responsiveness. Thus to avoid things like that:
```
You are about to permanently
delete the quest « Get a Pikachu
 ».
```